### PR TITLE
Superset link

### PR DESCRIPTION
**Pull request by yakloinsteak**
## Description

Adds a link to superset on the oauth (doorkeeper) admin page.

## Type of change
- [x ] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/views/doorkeeper/applications/index.html.erb
Comment (eanders) [inline app/views/doorkeeper/applications/index.html.erb:1]: Any reason this is in ERB?  The rest of the app is in haml.

Comment (eanders) [reply app/views/doorkeeper/applications/index.html.erb:1]: Oh, this is an override of the default from the gem maybe?

Comment (yakloinsteak) [reply app/views/doorkeeper/applications/index.html.erb:1]: It's pulled from the doorkeeper gem, so I went with making diffing against the original view easier. Want me to convert to haml?

Comment (eanders) [reply app/views/doorkeeper/applications/index.html.erb:1]: If you think that'd work, yeah, having that in haml will make our lives easier long-term.  Thanks!

**Pull request merged**